libraries/cli/include/cli/default_config.hpp

```diff
@@ -16,6 +16,7 @@ const char *default_json = R"foo({
   "network_max_peer_count": 50,
   "network_sync_level_size": 10,
   "network_packets_processing_threads": 10,
+  "network_peer_blacklist_timeout" : 600,
```

Comment:
I am not sure, but maybe we should have a config migration for this. I believe it is already merged to develop.
But maybe it is better to do it in PR to testnet\mainnet branch.

Reply:
this is an optional parameter which will default to this value if it is missing so I don't think we need to have config migration

Reply:
@kstdl what ? PR to testnet/mainnet branch ? why ?

Reply:
I would like to merge this before reseting the devnet so I am resolving this conversation. If this is something unresolved please create an issue for it